### ISSUE TITLE
Datasets vs Project

### ISSUE THREAD

**Issue by tischi** [mobie/mobie.github.io#38]
`https://github.com/mobie/yeast-clem-datasets`
I think I would prefer now 
`https://github.com/mobie/yeast-clem-project`

**Comment by constantinpape** [comment]
I agree. Let's change the names; old links will stay valid so this is not a problem,
